fda/grid.py

```diff
@@ -186,7 +186,13 @@ def __init__(self, data_matrix, sample_points=None,
         self.dataset_label = dataset_label
         self.axes_labels = axes_labels
         self.keepdims = keepdims
-        self.interpolator = interpolator
+
+        if interpolator is None:
```

Comment:
I continue thinking this logic belongs to the setter ;-)

Reply:
Changed in 9181b02.

Reply:
Changed in 9181b02.